### PR TITLE
overlord, o/devicestate: use a single test helper for resetting to a post boot state

### PR DESCRIPTION
**Pull request by bboozzoo**
Followup to https://github.com/snapcore/snapd/pull/10361 I promised to @pedronis

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thank you for this cleanup

File: overlord/devicestate/devicemgr.go [hunk 1]
```diff
@@ -1789,6 +1776,8 @@ func (h fdeSetupHandler) Error(err error) error {
 
 // ResetToPostBootState is only useful for integration testing.
 func (m *DeviceManager) ResetToPostBootState() {
```
Comment (pedronis) [inline overlord/devicestate/devicemgr.go:1778]: I would move this to after Ensure, that seems the most reasonable place I think

**Review by pedronis — APPROVED** [review]
thanks, a positioning nitpick now that I looked at code more closely again

File: overlord/devicestate/devicemgr.go (hunk 1 above)
Comment (bboozzoo) [reply overlord/devicestate/devicemgr.go]: moved

**Pull request merged**